Marlin/src/gcode/feature/ft_motion/M493.cpp

```diff
@@ -423,15 +416,8 @@ void GcodeSuite::M493() {
 
   #endif // HAS_Y_AXIS
 
-  planner.synchronize();
-
   if (flag.update) ftMotion.update_shaping_params();
```

Comment:
Seems like a smart idea to synchronize before modifying the parameters that `ftMotion` uses to do its motion. Are we certain it is / will always be safe to call `update_shaping_params` while a move is still ongoing?